### ISSUE TITLE
Bug when extending classes with 'spec.protoToAssign' optional in v5.8.5

### ISSUE THREAD

**Issue by glued** [babel/babel#2071]
geometry.js (example)
```
class Polygon{
  constructor(width, height){
    this.height = height;
    this.width = width;
  }
}

export class Square extends Polygon{
  constructor(length) {
    super(length, length);
  }
}
```
stack trace
```
Browserify Error 
/Users/glue/Desktop/babel-proto/public/js/geometry.js:11
function _inherits(subClass, superClass) { if (typeof superClass !== "function" && superClass !== null) { throw new TypeError("Super expression must either be null or a function, not " + typeof superClass); } subClass.prototype = Object.create(superClass && superClass.prototype, { constructor: { value: subClass, enumerable: false, writable: true, configurable: true } }); if (superClass) Object.setPrototypeOf ? Object.setPrototypeOf(subClass, superClass) : ( = subClass, _defaults(superClass)); }
                                                                                                                                                                                                                                                                                                                                                                                                                                                                              ^
ParseError: Unexpected token
```
gulp:
```
gulp.task('babelCompile', function() {
 	browserify('./public/js/script.js', { debug: true })
		.add(require.resolve('babel/polyfill'))
		.transform(babelify.configure({ optional: ['spec.protoToAssign']}))
		.bundle()
		.on('error', util.log.bind(util, 'Browserify Error'))
		.pipe(source('script.js'))
		.pipe(buffer())
		.pipe(sourcemaps.init({loadMaps: true}))
		.pipe(uglify({ mangle: false }))
		.pipe(sourcemaps.write('./'))
		.pipe(gulp.dest('./public/build'));
});
```

**Comment by sebmck** [comment]
Fixed as of `babel-plugin-proto-to-assign@1.0.4`. A full reinstall of Babel should fix this (just delete `node_modules` and `npm install` again). Thanks! :blush:

**Comment by glued** [comment]
so fast! verified fixed thanks again :+1:
